package.json

```diff
@@ -25,7 +25,7 @@
     "test:backend": "nyc --reporter=lcovonly mocha --reporter=min test/backend",
     "test:frontend": "cross-env NODE_ENV=development node test/frontend/runner.js",
     "test:report": "nyc report --reporter=html",
-    "test-integration": "cross-env NODE_ENV=development wdio test/wdio.docker.conf.js",
+    "test-integration": "cross-env NODE_ENV=development ANDROID=1 wdio test/wdio.local.conf.js",
```

Comment:
Is leaving `ANDROID=1` here intentional? We should be testing the main site here, no?

Reply:
Not intentional, I was experimenting. I'll remove this.